SMLHelper/Patchers/CraftDataPatcher.cs

```diff
@@ -131,6 +132,18 @@ private static void PreparePrefabIDCachePostfix()
 
         private static void AddCustomTechDataToOriginalDictionary()
         {
+            if (DuplicateTechDataAttempts.Count > 0)
+            {
+                Logger.Log($"Removing conflicting TechData entries from patching.{Environment.NewLine}" +
```

Comment:
Couldn't you use "\n"?

Reply:
It's generally better to always use `Environment.NewLine` for best compatibility.
This will use the correct line-break character for the file appropriate the OS and encoding,  
whether that be `\n` (Unix style) or `\r\n` (Windows Style).

The `output_log` file uses CRLF (`\r\n`) on Windows but it's possible that it could be using LF (`\n`) on the Mac version (or potentially someone running this on Linux using the new Proton API).  
And as a general rule, you don't want to have mixed line-ending styles in a single file.

So that's why it's generally a good habit to let .NET's `Environment.NewLine` deal with the line-endings for you.